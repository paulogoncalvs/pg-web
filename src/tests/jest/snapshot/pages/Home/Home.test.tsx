import { pageTests } from '..';

pageTests({
    name: 'Home',
    route: '/',
});
