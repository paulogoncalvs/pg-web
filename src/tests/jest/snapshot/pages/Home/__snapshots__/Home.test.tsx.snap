// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[PAGE] Home Page should render 1`] = `
<main>
  <header class="container p-4 py-10 mx-auto">
    <div class="flex items-center justify-between">
      <div class="flex items-center">
        <select class="py-1 text-sm text-zinc-900 dark:text-zinc-200 bg-white dark:bg-zinc-900 border-zinc-800 dark:border-zinc-200 border-2 font-sans ml-2 mr-2"
                aria-label="Language selection"
        >
          <option value="en"
                  selected
          >
            EN
          </option>
          <option value="pt">
            PT
          </option>
        </select>
      </div>
      <div class="flex justify-end">
        <a href="https://www.twitter.com/paulogoncalvs"
           aria-label="Twitter (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _hdr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="24"
               height="24"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://www.github.com/paulogoncalvs"
           aria-label="GitHub (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _hdr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="24"
               height="24"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://pt.linkedin.com/in/paulogoncalvs"
           aria-label="LinkedIn (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _hdr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="24"
               height="24"
          >
            <use href="#">
            </use>
          </svg>
        </a>
      </div>
    </div>
  </header>
  <div class="container px-6 pt-8 pb-8 mx-auto sm:pb-16">
    <div class="flex flex-col items-center">
      <h1 class="text-3xl font-black text-center opacity-0 sm:my-2 sm:text-5xl animate-fade-in-up-1">
        PAULO GONÃ‡ALVES
      </h1>
      <h2 class="text-2xl text-center lowercase opacity-0 sm:text-4xl animate-fade-in-up-2 pb-14">
        Front-End Engineer
      </h2>
      <div class="flex flex-col items-center sm:pt-14 pb-14 sm:flex-row">
        <div class="flex-shrink-0 opacity-0 animate-fade-in-dw-3">
          <img width="180"
               height="180"
               class="bg-zinc-200 dark:bg-zinc-800 lazy animate-pulse rounded-full"
               alt
               src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          >
        </div>
        <div class="max-w-xs text-sm opacity-0 sm:pl-14 space-y-4 sm:text-base sm:max-w-md pt-14 sm:pt-0 animate-fade-in-dw-4">
          <p>
            Born in Vila Nova de Gaia, Portugal.
          </p>
          <p class="mux">
            Currently working as a
            <strong>
              Front-End Engineering Manager
            </strong>
            /
            <strong>
              Lead Front-End Engineer
            </strong>
            at Porto Tech Center - Jumia.
          </p>
          <p>
            Check the
            <a href="https://pt.linkedin.com/in/paulogoncalvs"
               target="_blank"
               rel="noopener noreferrer"
               class="underline"
            >
              LinkedIn profile
            </a>
            for more information.
          </p>
        </div>
      </div>
      <div class="pb-8 opacity-0 sm:pt-8 animate-fade-in-dw-5">
        <h2 class="pb-4 text-base font-bold sm:text-xl">
          Wanna reach out?
        </h2>
        <a href="mailto:contact@paulogoncalves.dev"
           class="btn _i _prim"
        >
          <svg aria-hidden="true"
               class
               width="24"
               height="24"
          >
            <use href="#">
            </use>
          </svg>
          <span>
            E-mail me
          </span>
        </a>
      </div>
    </div>
  </div>
</main>
<footer>
  <div class="text-center bg-zinc-100 dark:bg-zinc-800">
    <div class="container flex flex-col items-center px-6 pt-16 mx-auto pb-14">
      <div class="pb-6 text-sm sm:text-base">
        This website was built from scratch with:
      </div>
      <div class="flex flex-wrap justify-center align-middle">
        <a href="https://www.typescriptlang.org/"
           aria-label="TypeScript (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://preactjs.com/"
           aria-label="Preact (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://webpack.js.org/"
           aria-label="Webpack (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://yarnpkg.com/"
           aria-label="Yarn (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://eslint.org/"
           aria-label="ESLint (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://stylelint.io/"
           aria-label="StyleLint (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://developers.google.com/web/tools/workbox"
           aria-label="Workbox (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://analytics.google.com/"
           aria-label="Google Analytics (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="28"
               height="28"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://tailwindcss.com/"
           aria-label="Tailwind CSS (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://playwright.dev/"
           aria-label="Playwright (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://jestjs.io/"
           aria-label="Jest (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://prettier.io/"
           aria-label="Prettier (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://postcss.org/"
           aria-label="PostCSS (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://www.deque.com/axe/"
           aria-label="Axe (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://sass-lang.com/"
           aria-label="Sass (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="32"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
        <a href="https://babeljs.io/"
           aria-label="Babel (opens in a new window)"
           target="_blank"
           rel="noopener noreferrer"
           class="ic-link _ftr"
        >
          <svg aria-hidden="true"
               class="fill-current"
               width="60"
               height="32"
          >
            <use href="#">
            </use>
          </svg>
        </a>
      </div>
    </div>
  </div>
  <div class="container flex flex-col items-center py-8 mx-auto text-xs text-center sm:py-16 text-zinc-500">
    <p>
      optimized for modern browsers
    </p>
    <p class="pt-2">
      paulogoncalves.dev Â© 2021  - 2022 ðŸ™‚
    </p>
  </div>
</footer>
`;
